packages/browser-agent-core/README.md

```diff
@@ -1,7 +1,248 @@
 [![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
 
-# New Relic Browser Agent Core
+# New Relic Browser Agent
 
-**See [@newrelic/browser-agent](https://www.npmjs.com/package/@newrelic/browser-agent) package**
+The New Relic Browser Agent instruments your website and provides observability into the performance and behavior of your application. This NPM Library is an ***in-progress*** implementation of the New Relic Browser Agent, accessible through `NPM`. Please see the [Differences](#differences) and [Features](#features) sections to compare how this library differs from the other offerings of the New Relic Browser Agent.
 
-This package contains the modules used by the New Relic Browser agent NPM package, this is used internally only and is not intended for use as a public-facing library.
+## Installation
```

Comment:
As of right now, these readme instructions are not relevant for this package (@newrelic/browser-agent was intended to be the customer interface, while @newrelic/browser-agent-core was intended to just be internal building blocks)